org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/internal/ZigBeeNetworkStateSerializerImpl.java

```diff
@@ -180,4 +180,16 @@ public void deserialize(final ZigBeeNetworkManager networkState) {
         logger.debug("Loading ZigBee network state: Done.");
     }
 
+    public synchronized void delete() {
```

Comment:
Suggestion: Add javadoc comment to `delete` method (also to stay consistent with the remainder of the class).

Reply:
Thx. Totally forgot to add the javadoc.